src/features/checker/components/ProjectHeader/ProjectHeader.tsx

```diff
@@ -0,0 +1,22 @@
+import { Avatar } from "@/primitives/Avatar";
+import { BannerImage } from "@/primitives/BannerImage";
+
+import { Project } from "./types";
+
+export interface ProjectHeaderProps {
+  project: Project;
+}
+
+export const ProjectHeader = ({ project }: ProjectHeaderProps) => {
+  return (
+    <div className="relative ">
```

Comment:
This div should cover the avatar as well, I left a suggestion based on the selected sizes of the avatar.
100% would be the banner, and we add half of the height of the avatar.
```suggestion
    <div className="relative h-[calc(100%+48px)] md:h-[calc(100%+96px)]">
```